modules/local/append_clusters/main.nf

```diff
@@ -30,7 +30,7 @@ process APPEND_CLUSTERS {
     init_splits=\$(head -n 1 initial-cluster-address.txt | awk -F '${params.gm_delimiter}' '{print NF}')
     add_splits=\$( head -n 1 additional-cluster-address.txt | awk -F '${params.gm_delimiter}' '{print NF}')
 
-    if [ "\$init_splits" != "\$add_splits" ]; then
+    if [ "\$init_splits" != "\$add_splits" ] && [ "\$init_splits" != "" ]; then
```

Comment:
This code basically checks that the number of delimiters is the same between files. The addition changes this so that an empty "initial clusters" file (derived from the sample sheet) doesn't cause a program error. The rest of the code appends this empty file to the "additional clusters" file.